twitter/Views/TweetCell.m

```diff
@@ -7,18 +7,130 @@
 //
 
 #import "TweetCell.h"
+#import "APIManager.h"
+#import "DateTools.h"
+
+@interface TweetCell()
+
+@property (weak, nonatomic) IBOutlet UIImageView *profilePicture;
```

Comment:
This is an interesting one. I know why you did this and you were probably encouraged to but I disagree with the decision.

Views should be dumb. They should just be told what to display. The pattern you had before where the controller told the view what to display based on the tweet was correct imo. Even better would be a separate type that is just responsible for decorating the cell. It would know how to decorate a cell using a tweet and that would be its only responsibility.